src/zkevm_specs/evm_circuit/instruction.py

```diff
@@ -575,9 +575,11 @@ def mul_word_by_u64(self, multiplicand: Word, multiplier: Expression) -> Tuple[W
             (multiplicand_hi * multiplier.expr() + quotient_lo).n, 1 << 128
         )
 
+        self.constrain_zero(FQ(quotient_hi))
+
         product_bytes = product_lo.to_bytes(16, "little") + product_hi.to_bytes(16, "little")
 
-        return Word((FQ(product_lo), FQ(product_hi))), FQ(quotient_hi)
+        return Word(product_bytes)
```

Comment:
The signature of the Word is two cells.
```suggestion
        return Word((FQ(product_lo), FQ(product_hi)))
```

Reply:
Oh, I see. Applied.